### PR TITLE
quincy: monitoring: several Grafana fixes

### PR DESCRIPTION
**Pull request by epuertat**
* #45533
* #45578
* #45583
* #45335
* #45254
* #45739
* #45763
* #45793
* #45187
* #45647

Fixes: https://tracker.ceph.com/issues/55200
Fixes: https://tracker.ceph.com/issues/55201
Fixes: https://tracker.ceph.com/issues/55128
Fixes: https://tracker.ceph.com/issues/55209
Fixes: https://tracker.ceph.com/issues/55210
Fixes: https://tracker.ceph.com/issues/55211
Fixes: https://tracker.ceph.com/issues/55120

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [x] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [x] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by epuertat** [comment]
jenkins test make check

**Comment by epuertat** [comment]
jenkins test windows

**Comment by epuertat** [comment]
jenkins retest this please

**Comment by epuertat** [comment]
jenkins test dashboard

**Comment by epuertat** [comment]
jenkins test docs

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -48,10 +48,10 @@ FuncT = TypeVar('FuncT', bound=Callable)
...
 DEFAULT_IMAGE_IS_MASTER = False
 DEFAULT_IMAGE_RELEASE = 'quincy'
-DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
-DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v0.18.1'
-DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.20.0'
-DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:6.7.4'
+DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.33.4'
+DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.3.1'
+DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.23.0'
+DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:8.3.5'
```
Comment (sunilangadi2) [inline src/cephadm/cephadm:54]: @epuertat changes look good to me,
Are we planning to backport this upgraded grafana and monitoring stack versions to pacific as well?

**Review by sunilangadi2 — APPROVED** [review]

**Review by aaSharma14 — APPROVED** [review]

File: src/cephadm/cephadm (hunk 1 above)
Comment (epuertat) [reply src/cephadm/cephadm:54]: @sunilangadi2 potentially any user could have this in Pacific just by updating their DEFAULT_GRAFANA_IMAGE to this version (as it also includes the embedded Grafana Dashboards).

In any case @aaSharma14 is working to remove the embedded Grafana Dashboards from the container, and use a vanilla Grafana container (no more custom upstream/downstream builds of ceph-grafana container). Please @aaSharma14 add the trackers for these work items.

Comment (aaSharma14) [reply src/cephadm/cephadm:54]: https://tracker.ceph.com/issues/53275

**Comment by epuertat** [comment]
@yuriw given it passed CI and Dashboard teuthology runs I'll proceed and merge this one.

**Comment by epuertat** [comment]
> @yuriw given it passed CI and Dashboard teuthology runs I'll proceed and merge this one.

Ahh, not, I can't (6 reviewers required). So it's all yours @yuriw 😉

**Pull request merged**